docs/Tutorials/CCE.md

```diff
@@ -108,102 +194,50 @@ remaining values are the complex modes in m-varies-fastest format. That is,
 "Lapse_Re(2,-2)", "Lapse_Im(2,-2)"
 ```
 
-#### Spherical harmonic nodes {#cartesian_spherical_nodes}
+Each dataset in the file must also have an attribute
+named `Legend` which is an ASCII-encoded null-terminated variable-length string.
 
-\warning This format is not yet fully supported but is under development. If you
-need it please file an issue so we can escalate the priority.
+##### Spherical harmonic nodes {#spherical_nodes}
 
-In this format the value of the functions at specially chosen collocation points
-(grid points) is read in. This allows SpECTRE to perform integrals, derivatives,
-and interpolation exactly on the input data. These grid points are
-Gauss-Legendre in $cos(\theta)$ and equally spaced in $\phi$. Below is a routine
-for computing the spherical harmonic $\theta$ and $\phi$ values. These can be
-used to compute the Cartesian locations for a given radius using the standard
-transformation. The routine supports \f$\ell\in[4, 32]\f$.
+In the metric nodal format, the value of the functions at specially chosen
+collocation points (grid points) is read in. This allows SpECTRE to perform
+integrals, derivatives, and interpolation exactly on the input data. These grid
+points are Gauss-Legendre in $cos(\theta)$ and equally spaced in $\phi$. Below
+is a routine for computing the spherical harmonic $\theta$ and $\phi$ values.
+These can be used to compute the Cartesian locations for a given radius using
+the standard transformation. The routine supports \f$\ell\in[4, 32]\f$.
```

Comment:
Should we note (maybe even `\note`) that this is likely the easiest format for people to write out?